.github/workflows/benchmarks.yaml

```diff
@@ -0,0 +1,58 @@
+name: Benchmarks
+
+env:
+  PY_COLORS: 1
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/python-tests.yaml
+      - "**/*.py"
+      - requirements.txt
+      - requirements-dev.txt
+      - setup.cfg
+      - Dockerfile
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  run-benchmarks:
+    name: Benchmark
+    # Runs with ephemeral API and SQLite database right now
```

Comment:
Is using SQLite just the easiest thing to do or do we want to make sure we have good benchmarks against SQLite _and_ Postgres at some point?

Reply:
I think testing against both and including tests with a hosted API instead of ephemeral are all important, but I think they'll come after this PR.